ping.h

```diff
@@ -28,7 +28,7 @@ struct ping_resp {
 };
 
 bool ping_start(struct ping_option *ping_opt);
-void ping(const char *name, int count, int interval, int size, int timeout);
+//void ping(const char *name, int count, int interval, int size, int timeout);
```

Comment:
We will not remove this.
This is the standard function signature and it's the same one used by  esp8266.